content/actions/learn-github-actions/workflow-syntax-for-github-actions.md

```diff
@@ -1051,7 +1051,8 @@ Public actions may specify expected environment variables in the README file. If
 steps:
   - name: My first action
     env:
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # note: github secrets may not begin with `GITHUB_`
```

Comment:
I can see why you added the note, but I think you probably wouldn't have thought to prefix the token name with `GITHUB_` if we hadn't chosen a poor example here. I think that now you've fixed the example, this note is unnecesary.

```suggestion
```